Lib/platform.py

```diff
@@ -9,8 +9,9 @@
 
 """
 #    This module is maintained by Marc-Andre Lemburg <mal@egenix.com>.
-#    If you find problems, please submit bug reports/patches via the
-#    Python bug tracker (http://bugs.python.org) and assign them to "lemburg".
+#    If you find problems, please submit bug reports/patches as detailed
+#    in the Python devguide (https://devguide.python.org/tracker/) and
+#    assign them to "lemburg".
```

Comment:
"lemburg" should be "malemburg" here.

Reply:
@ronaldoussoren Thanks for pointing that out correction being pushed in a few minutes.